utils/post.js

```diff
@@ -156,14 +156,15 @@ const modifyAnonymousAndBlockPost = async (
   return feedWithAnonymous;
 };
 
-const modifyAnonimityPost = (item) => {
+const modifyAnonimityPost = async (item, isBlurredPost = true) => {
   const newItem = {...item};
 
   if (newItem.anonimity) {
     newItem.actor = {};
     newItem.to = [];
     newItem.origin = null;
     newItem.object = '';
+    newItem.isBlurredPost = isBlurredPost;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyAnonimityPost` function has been updated to include a new parameter `isBlurredPost` with a default value of `true`. This parameter is used to set the `isBlurredPost` property of the `newItem` object. However, this property is only set when `newItem.anonimity` is true. If `newItem.anonimity` is false, the `isBlurredPost` property will not be set on the `newItem` object, which could potentially lead to undefined behavior elsewhere in the code where this property is expected to exist.

To avoid potential issues, it would be better to always set the `isBlurredPost` property on the `newItem` object, regardless of the value of `newItem.anonimity`.

```diff
-  if (newItem.anonimity) {
-    newItem.actor = {};
-    newItem.to = [];
-    newItem.origin = null;
-    newItem.object = '';
-    newItem.isBlurredPost = isBlurredPost;
-  }
+  newItem.isBlurredPost = isBlurredPost;

+  if (newItem.anonimity) {
+    newItem.actor = {};
+    newItem.to = [];
+    newItem.origin = null;
+    newItem.object = '';
+  }

   return newItem;
```
This way, `isBlurredPost` is always defined on `newItem`, ensuring consistent behavior across the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->